### PR TITLE
Minor doc fix

### PR DESCRIPTION
**Pull request by containscafeine**

### CODE REVIEW

**Comment by surajssd** [comment]
@containscafeine please sign linux foundation cla, otherwise fix LGTM

**Comment by containscafeine** [comment]
@surajssd just did!

File: docs/development.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ If it is not a valid version try, make sure you have updated the godep repo with
 1. Populate dependencies for your Kompose.
 
 ```console
-cd $GOPATH/src/github.com/kubernetes-incubator
+cd $GOPATH/src/github.com/kubernetes-incubator/kompose/
```
Comment (cdrage) [inline docs/development.md]: remove the trailing / :)

**Comment by cdrage** [comment]
@sebgoa revert this? trailing `/` still there..

**Comment by cdrage** [comment]
@sebgoa seems this was merged before the trailing `/` was removed. opened up https://github.com/kubernetes-incubator/kompose/pull/305

**Comment by sebgoa** [comment]
what's wrong with the trailing / ?

**Comment by cdrage** [comment]
@sebgoa nothing, just OCD: http://unix.stackexchange.com/questions/212805/should-i-use-a-slash-at-the-end-of-path-variables-in-shell-script-or-not

**Pull request merged**